// @angular/forms/forms.ngfactory.ts
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties,missingOverride}
 */
 /* tslint:disable */


import * as import0 from '@angular/core';
import * as import1 from '@angular/forms';
class FormsModuleInjector extends import0.ɵNgModuleInjector<import1.FormsModule> {
  _ɵba_0:import1.ɵba;
  _FormsModule_1:import1.FormsModule;
  __ɵi_2:import1.ɵi;
  _LOCALE_ID_3:any;
  _TRANSLATIONS_FORMAT_4:any;
  constructor(parent:import0.Injector) {
    super(parent,([] as any[]),([] as any[]));
  }
  get _ɵi_2():import1.ɵi {
    if ((this.__ɵi_2 == null)) { (this.__ɵi_2 = new import1.ɵi()); }
    return this.__ɵi_2;
  }
  createInternal():import1.FormsModule {
    this._ɵba_0 = new import1.ɵba();
    this._FormsModule_1 = new import1.FormsModule();
    this._LOCALE_ID_3 = 'zh';
    this._TRANSLATIONS_FORMAT_4 = 'xlf';
    return this._FormsModule_1;
  }
  getInternal(token:any,notFoundResult:any):any {
    if ((token === import1.ɵba)) { return this._ɵba_0; }
    if ((token === import1.FormsModule)) { return this._FormsModule_1; }
    if ((token === import1.ɵi)) { return this._ɵi_2; }
    if ((token === import0.LOCALE_ID)) { return this._LOCALE_ID_3; }
    if ((token === import0.TRANSLATIONS_FORMAT)) { return this._TRANSLATIONS_FORMAT_4; }
    return notFoundResult;
  }
  destroyInternal():void {
  }
}
export const FormsModuleNgFactory:import0.NgModuleFactory<import1.FormsModule> = new import0.NgModuleFactory<any>(FormsModuleInjector,import1.FormsModule);
class ɵbaInjector extends import0.ɵNgModuleInjector<import1.ɵba> {
  _ɵba_0:import1.ɵba;
  _LOCALE_ID_1:any;
  _TRANSLATIONS_FORMAT_2:any;
  constructor(parent:import0.Injector) {
    super(parent,([] as any[]),([] as any[]));
  }
  createInternal():import1.ɵba {
    this._ɵba_0 = new import1.ɵba();
    this._LOCALE_ID_1 = 'zh';
    this._TRANSLATIONS_FORMAT_2 = 'xlf';
    return this._ɵba_0;
  }
  getInternal(token:any,notFoundResult:any):any {
    if ((token === import1.ɵba)) { return this._ɵba_0; }
    if ((token === import0.LOCALE_ID)) { return this._LOCALE_ID_1; }
    if ((token === import0.TRANSLATIONS_FORMAT)) { return this._TRANSLATIONS_FORMAT_2; }
    return notFoundResult;
  }
  destroyInternal():void {
  }
}
export const ɵbaNgFactory:import0.NgModuleFactory<import1.ɵba> = new import0.NgModuleFactory<any>(ɵbaInjector,import1.ɵba);
class ReactiveFormsModuleInjector extends import0.ɵNgModuleInjector<import1.ReactiveFormsModule> {
  _ɵba_0:import1.ɵba;
  _ReactiveFormsModule_1:import1.ReactiveFormsModule;
  __FormBuilder_2:import1.FormBuilder;
  __ɵi_3:import1.ɵi;
  _LOCALE_ID_4:any;
  _TRANSLATIONS_FORMAT_5:any;
  constructor(parent:import0.Injector) {
    super(parent,([] as any[]),([] as any[]));
  }
  get _FormBuilder_2():import1.FormBuilder {
    if ((this.__FormBuilder_2 == null)) { (this.__FormBuilder_2 = new import1.FormBuilder()); }
    return this.__FormBuilder_2;
  }
  get _ɵi_3():import1.ɵi {
    if ((this.__ɵi_3 == null)) { (this.__ɵi_3 = new import1.ɵi()); }
    return this.__ɵi_3;
  }
  createInternal():import1.ReactiveFormsModule {
    this._ɵba_0 = new import1.ɵba();
    this._ReactiveFormsModule_1 = new import1.ReactiveFormsModule();
    this._LOCALE_ID_4 = 'zh';
    this._TRANSLATIONS_FORMAT_5 = 'xlf';
    return this._ReactiveFormsModule_1;
  }
  getInternal(token:any,notFoundResult:any):any {
    if ((token === import1.ɵba)) { return this._ɵba_0; }
    if ((token === import1.ReactiveFormsModule)) { return this._ReactiveFormsModule_1; }
    if ((token === import1.FormBuilder)) { return this._FormBuilder_2; }
    if ((token === import1.ɵi)) { return this._ɵi_3; }
    if ((token === import0.LOCALE_ID)) { return this._LOCALE_ID_4; }
    if ((token === import0.TRANSLATIONS_FORMAT)) { return this._TRANSLATIONS_FORMAT_5; }
    return notFoundResult;
  }
  destroyInternal():void {
  }
}
export const ReactiveFormsModuleNgFactory:import0.NgModuleFactory<import1.ReactiveFormsModule> = new import0.NgModuleFactory<any>(ReactiveFormsModuleInjector,import1.ReactiveFormsModule);
